tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml

```diff
@@ -59,6 +59,11 @@ spec:
           COMPONENT=$(jq -c --argjson i "$i" '.components[$i]' "${PYXIS_FILE}")
           IMAGEURL=$(jq -r '.containerImage' <<< "${COMPONENT}")
           NUM_PYXIS_IMAGES=$(jq '.pyxisImages | length' <<< "${COMPONENT}")
+          # Create a docker config file with just one entry for the image as partial oci matches
+          # result in authenticated errors
+          DOCKER_CONFIG="$(mktemp -d)"
+          export DOCKER_CONFIG
```

Comment:
You could move these two lines just above the `for` loop - no need to create a new dir each time. But this is acceptable as well.

Reply:
fixed